### PR TITLE
Php8 attributes fix

### PR DESCRIPTION
**Pull request by mathieupetrini**
Fix PHP 8 Attribute reader

### CODE REVIEW

File: src/Metadata/Driver/AttributeReader.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+namespace Vich\UploaderBundle\Metadata\Driver;
+
+use Doctrine\Common\Annotations\Reader;
+use ReflectionAttribute;
+use ReflectionClass;
+use ReflectionMethod;
+use ReflectionProperty;
+use Vich\UploaderBundle\Mapping\AnnotationInterface;
+
+class AttributeReader implements Reader
```
Comment (garak) [inline src/Metadata/Driver/AttributeReader.php:12]: Can we make this class final?

File: tests/Metadata/Driver/AttributeReaderTest.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+namespace Vich\UploaderBundle\Tests\Metadata\Driver;
+
+use PHPUnit\Framework\TestCase;
+use Vich\UploaderBundle\Mapping\Annotation\Uploadable;
+use Vich\UploaderBundle\Mapping\Annotation\UploadableField;
+use Vich\UploaderBundle\Metadata\Driver\AttributeReader;
+use Vich\UploaderBundle\Tests\DummyAttributeEntity;
+
+class AttributeReaderTest extends TestCase
```
Comment (garak) [inline tests/Metadata/Driver/AttributeReaderTest.php:11]: This needs to be final

**Review by garak — CHANGES_REQUESTED** [review]

**Review by garak — APPROVED** [review]

**Comment by redecs** [comment]
Thank you @mathieupetrini for adding the attributes support and @garak for reviewing. Looking forward to seeing this in a realse! 🎉

**Comment by adpeyre** [comment]
Thanks @mathieupetrini !
It works for me with the dev-master version.

@garak A release soon ? :)

**Comment by garak** [comment]
> Thanks @mathieupetrini ! It works for me with the dev-master version.

Don't use dev-master. Use our alias, customized with the current stable version +1 (currently, you can use 1.19-dev).
In this way, you can get the latest release now _and_ the stable release as soon as it will be out.
 
> @garak A release soon ? :)

There's no ETA at the moment.

**Comment by adpeyre** [comment]
@garak The 1.19-dev version doesn't exist. ;)

**Pull request merged**